### PR TITLE
fix: replace regex with string operations

### LINKED ISSUE #11: Error when connecting to the client
I've attempted to use this crate in my Windows10 app and it worked at first, but I found that sometimes it would not connect to the client and only way to fix that would be to reset client (sometimes multiple times) until the app was able to connect.

I've cloned the repo and ran the example, and I get the following error:
```
thread 'main' panicked at examples\connect_to_client.rs:8:50:
called `Result::unwrap()` on an `Err` value: Tungstenite(Http(Response { status: 401, version: HTTP/1.1, headers: {"www-authenticate": "Basic realm=\"RiotRemoting\"", "content-length": "0"}, body: Some([]) }))
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\examples\connect_to_client.exe` (exit code: 101)
```
The error persists when re-runing the example.
I've tried using https://github.com/BlossomiShymae/Needlework.Net at the same time that this crate give me the error, and needleowork.net works, so I assume that there is a bug somewhere in the connection process.
I can't find a consistent way to reprodcue the error. Only thing I found is that if the error is going to happen with one client instance it will happen for as long as the client is running, same thing is true when there is no error, until you restart the client you will get the same result.

### PR DESCRIPTION
**Pull request by brkicdusan**
Resolves #11

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -176,23 +176,23 @@ fn from_process(process: &str) -> Option<Vec<String>> {
...
-
-    Ok((token, port))
+    let args: Vec<&str> = value.split("\"").collect();
+
+    let mut token = Err(Error::AppNotRunning);
+    let mut port = Err(Error::AppNotRunning);
+
+    for arg in args {
+        if arg.starts_with("--remoting-auth-token") {
+            let val = arg.split("=").collect::<Vec<&str>>()[1];
```
Comment (coltiebaby) [inline src/client.rs:186]: mind getting rid of the index here? 

```rust
let val = arg.split("=").collect::<Vec<&str>>().first();
let token = val.ok_or(Error::AppNotRunning);
```

File: src/client.rs [hunk 2]
```diff
@@ -176,23 +176,23 @@ fn from_process(process: &str) -> Option<Vec<String>> {
...
+            token = Ok(val);
+        }
+        if arg.starts_with("--app-port") {
+            let val = arg.split("=").collect::<Vec<&str>>()[1];
```
Comment (coltiebaby) [inline src/client.rs:190]: same here.

**Review by coltiebaby — CHANGES_REQUESTED** [review]
LGTM for the most part. Just wanna avoid having a runtime error happen if we get by index here.

File: src/client.rs (hunk 1 above)
Comment (brkicdusan) [reply src/client.rs:186]: Yeah I'll change it no problem, though if I understand how args work it should always split into exactly 2 parts so this should still be fine I think.

Comment (coltiebaby) [reply src/client.rs:186]: you're probably right. This is just the only thing that's bitten us at work lately so i try to avoid it as much as possible.

**Comment by coltiebaby** [comment]
Cool thanks!

File: src/client.rs (hunk 1 above)
Comment (brkicdusan) [reply src/client.rs]: Now that I changed it, it looks cleaner without indexing, plus it avoids needing to make a vector. I'm kind of new to rust still so I overcomplicate stuff like this a lot.

**Pull request merged**